_posts/2022-02-18-ols-is-hiring.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Job vacancy: Community Researcher and Programme Coordinator - OLS is hiring!"
+authors:
+- bebatut
+- emmyft
+- malvikasharan
+- yochannah
+image: ../images/2022-02-18-for-hire.jpg
+photos:
+  name: Photo by Clem Onojeghuo on Unsplash
+  url: https://unsplash.com/@clemono
+---
+
+OLS is hiring a Community Researcher and Programme Coordinator.
+
+_Note: If you like the scope of this job but unsure if your profile fits this role - please apply anyway. We are very happy to be contacted for general queries, please email jobs@openlifesci.org. You can also book a call with the Executive Director, Yo Yehudi, for specific questions about this vacancy: [book a 15 minute chat](https://calendly.com/yo-yehudi/ols-chat-with-yo?month=2022-02)._
+
+Open Life Science (OLS) is hiring an open, empathic, and creative colleague to coordinate  our cohort-based training and mentoring programmes and lead a research-based study to understand the long-term impacts of our work.
+
+Open Life Science (OLS, [https://openlifesci.org/](https://openlifesci.org/)) cultivates the next generation of researchers and community leaders through open science training and mentoring. Through 16-week long cohort-based programmes, OLS enables researchers to learn about, reflect on and integrate open science principles in their work. With the support of our community members, OLS is committed to delivering high quality training, mentoring and peer-based support to our community members.
+
+## Position Purpose
+
+This role encompasses two areas of OLS’ work: community management and community research. As a community manager, you will support the leadership team and OLS community in administration and communication tasks essential to smoothly run each cohort and share information in an open way. As a community researcher, you will help to carry out peer-reviewed research to understand transformational effects of open research training and mentoring interventions.
+## Key responsibilities
+- Community management and operational tasks involve communicating with the community members via Slack space, mailing lists and social media.
+- Lead a research-based study to assess the long-term impact of OLS’s programme on graduates and their communities.
+- 5-10% of the time for this position will be dedicated to supporting other open projects. This will include working in collaboration with community members, contributing to other open science projects on behalf of OLS and representing OLS in open science community events.
+- Implement, improve, and iterate on research practises for yourself and the OLS team, including ethics in research, version control, and open communications.
+
+## Essential selection criteria:
```

Comment:
```suggestion
## Essential selection criteria

```

Reply:
this shows as outdated so I can't merge it for some reason, will do manually.